src/main/scala/dxWDL/Main.scala

```diff
@@ -1,1119 +1,13 @@
 package dxWDL
 
-import com.typesafe.config._
-import java.nio.file.{Path, Paths}
-
-import spray.json.{JsString, _}
-import dxWDL.base._
-import dxWDL.compiler.Tree
-import dxWDL.dx._
-import dxWDL.util._
-
-import scala.collection.mutable
+import dx.core.util.MainUtils.normKey
 
+// Temporary main class to dispatch to either compiler or exec - until these are split into separate JARs.
```

Comment:
yes. In short we want to split it into two separate assemblies (JARs). The user will only ever deal with the compiler, and the runtime will only ever deal with exec. This will enable us to minimize the size of the runtime assembly.